src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -200,6 +211,14 @@ protected override void BeginProcessing()
                 Exception exception = new(TestJsonCmdletStrings.InvalidJsonSchema, e);
                 ThrowTerminatingError(new ErrorRecord(exception, "InvalidJsonSchema", ErrorCategory.InvalidData, resolvedpath));
             }
+
+            _documentOptions = new JsonDocumentOptions
+            {
+                CommentHandling = Options.Contains("IgnoreComments", StringComparer.OrdinalIgnoreCase)
+                    ? JsonCommentHandling.Skip
+                    : JsonCommentHandling.Disallow,
+                AllowTrailingCommas = Options.Contains("AllowTrailingCommas", StringComparer.OrdinalIgnoreCase)
```

Comment:
We use the string constants here and in ValidateSet. Why not follow the dry principle and define C# named constants?

Reply:
Fixed in cbbc834e8